pageserve/test_serve.py

```diff
@@ -18,9 +18,21 @@
 
 
 class TestServe(unittest.TestCase):
-    def test_get_auth(self):
-        # TODO: test authorization retrieval
-        pass
+    def test_auth_json(self):
+        """Checks if users service returns a correctly formatted object.
+
+        A dictionary with a boolean 'edit_access' field should be received.
+        """
+        response = app.get_auth_json('example_user')
+        valid_response = response['edit_access'] or not response['edit_access']
+        self.assertTrue(valid_response)
```

Comment:
Right now this test kinda just makes sure you don't get an AttributeError trying to access 'edit_access'.  response.json() from your get_auth_json function decodes json into a Python dict, so response['edit_access'] should be guaranteed to be a bool. As it is now, if response['edit_access'] is something else like a string, I think your code implicitly casts it to a bool and would probably pass. To fix this, you could just add another assert to make sure response['edit_access'] is an instance of bool.

Reply:
It seems like the defensive option would be to treat anything except `True` as `False`, using `response['edit_access'] is True` in `has_edit_access` (or rolling that into `get_auth_json` if you don't need to do anything else with the returned information from `get_auth_json`.

Reply:
I think we wanted to make the test independent of whether the given user actually has edit access in the db that the users service is deployed with, so a better comparison might be 
```python
self.assertTrue(response['edit_access'] is True or response['edit_access'] is False)
```

Reply:
The test should know whether `edit_access` should be `True` or `False` in this test.

Reply:
Changed test to reflect this in e163db6e3d8bc96d12dd6611fec67bdd3bbff5b9